src/components/common/Navbar/navbar.module.scss

```
@@ -0,0 +1,214 @@
+@import "src/styles/scaffold";
+
+.fs-navbar {
+  // --------------------------------------------------------
+  // Design Tokens for Navbar
+  // --------------------------------------------------------
+
+  // Default properties
+
+  --fs-navbar-padding: 0;
+
+  --fs-navbar-border-bottom-width                             : var(--fs-border-width);
+  --fs-navbar-border-bottom-color                             : var(--fs-color-neutral-2);
+
+  --fs-navbar-box-shadow                                      : 0 var(--fs-spacing-0) var(--fs-spacing-3) rgb(0 0 0 / 5%);
+
+  // Row
+  --fs-navbar-row-width                                       : 100%;
+
+  // Button Collapse (Search)
+  --fs-navbar-button-collapse-margin-left                     : calc(-1 * var(--fs-spacing-3));
+
+  // Button Cart
+  --fs-navbar-button-cart-notebook-margin-left                : var(--fs-spacing-3);
+
+  // Search Input Wrapper
+  --fs-navbar-search-input-wrapper-transition-property        : flex;
+  --fs-navbar-search-input-wrapper-transition-timing          : 0.3s;
+  --fs-navbar-search-input-wrapper-transition-function        : ease-in;
+
+  // Search
+  --fs-navbar-search-button-icon-mobile-width                 : var(--fs-spacing-5);
+  --fs-navbar-search-button-icon-mobile-height                : var(--fs-navbar-search-button-icon-mobile-width);
+
+  // Search variation - Expanded
+  --fs-navbar-search-expanded-width                           : 100%;
+
+  --fs-navbar-search-input-wrapper-expanded-margin-right      : 0.625rem;
+  --fs-navbar-search-input-wrapper-expanded-margin-left       : var(--fs-spacing-9);
+
+  --fs-navbar-search-input-expanded-width                     : calc(100% - var(--fs-spacing-7));
+  --fs-navbar-search-icon-expanded-margin-right               : 0;
+
+  --fs-navbar-search-button-icon-expanded-margin-right        : -4.063rem;
+
+  --fs-navbar-search-button-icon-expanded-transition-property : margin;
+  --fs-navbar-search-button-icon-expanded-transition-timing   : 0.3s;
+  --fs-navbar-search-button-icon-expanded-transition-function : ease-in;
+
+  // Header
+  --fs-navbar-header-padding-top                              : var(--fs-spacing-1);
+  --fs-navbar-header-padding-bottom                           : var(--fs-spacing-2);
+  --fs-navbar-header-overflow-x                               : clip;
+
+  --fs-navbar-header-notebook-padding-top                     : var(--fs-spacing-1);
+  --fs-navbar-header-notebook-padding-bottom                  : var(--fs-spacing-1);
+
+  // Logo
+  --fs-navbar-logo-width                                      : 11rem;
+  --fs-navbar-logo-height                                     : 2.375rem;
+  --fs-navbar-logo-min-height                                 : 2.375rem;
+  --fs-navbar-logo-padding                                    : 0;
+  --fs-navbar-logo-mobile-margin-right                        : var(--fs-spacing-5);
```

```suggestion
  --fs-navbar-logo-margin-right-mobile                        : var(--fs-spacing-5);
```

Same thing here.

Pls remember to update all the occurrences, ok?